node/node.go

```diff
@@ -677,7 +676,11 @@ func NewNode(config *cfg.Config,
 	}
 
 	// Determine whether we should attempt state sync.
-	stateSync := config.StateSync.Enable && !onlyValidatorIsUs(state, pubKey)
+	//
+	// If the validator has 1/3+ of the voting power, no new blocks were minted
+	// since 2/3+ of the voting power is required, hence we can safely skip fast
+	// and state sync.
+	stateSync := config.StateSync.Enable && !validatorHasMoreThan13Power(state, pubKey)
```

Comment:
I don't know if we should skip state sync here. This probably isn't likely to come up very often, but state sync will be explicitly enabled by the operator and only run if the node has no state, so if there are other nodes in the network I think we should attempt state sync regardless of how much voting power we have (otherwise it's impossible to use state sync to recover a validator).

Reply:
again, I think you're making an assumption that the network can progress with 1/3+ of the voting power being offline

Reply:
If the node doesn't have any local state, it only knows what power it had at genesis, not what power it has at the current chain head.